gallery/src/components/NxFormSelect/NxFormSelectPage.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxCode, NxTable, NxP, NxTextLink, NxTile, NxH3 } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxFormSelectExample from './NxFormSelectExample';
+
+const sourceCode = require('./NxFormSelectExample?raw');
+
+const NxFormSelectPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        <NxCode>NxFormSelect</NxCode> provides styling and basic behavior for <NxCode>&lt;select&gt;</NxCode>
+        elements. Formerly, this component was a simple "convenience component" that provided no extra capabilities
+        over the basic DOM element. However it has since been upgraded to a full component in order to support
+        pristine state tracking.
+      </NxP>
+      <NxTile.Subsection>
+        <NxTile.SubsectionHeader>
+          <NxH3>Props</NxH3>
+        </NxTile.SubsectionHeader>
+        <NxTable>
+          <NxTable.Head>
+            <NxTable.Row>
+              <NxTable.Cell>Prop</NxTable.Cell>
+              <NxTable.Cell>Type</NxTable.Cell>
+              <NxTable.Cell>Required</NxTable.Cell>
+              <NxTable.Cell>Default</NxTable.Cell>
+              <NxTable.Cell>Description</NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Head>
+          <NxTable.Body>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>isPristine</NxCode></NxTable.Cell>
+              <NxTable.Cell>boolean</NxTable.Cell>
+              <NxTable.Cell>No</NxTable.Cell>
+              <NxTable.Cell>false</NxTable.Cell>
+              <NxTable.Cell>
+                Should be set to true when the user has not yet adjusted the value of this form field.
```

Comment:
Why isn't the default of this "true"?

Reply:
> Why isn't the default of this "true"?

Why should it be?  I'd rather have a field that always looks non-pristine than one that always looks pristine.  I'd make this required but I don't want to break backwards compatibility (with the former convenience component) right now.

Reply:
> > Why isn't the default of this "true"?
> 
> Why should it be? I'd rather have a field that always looks non-pristine than one that always looks pristine. I'd make this required but I don't want to break backwards compatibility (with the former convenience component) right now.

I was thinking that the default state would be pristine because in almost all cases the select would be in a pristine state when the form is initially loaded. I'm ok with not making it the default to avoid breaking the backwards compatibility.